apps/parsley/src/utils/filterLogs/index.ts

```diff
@@ -11,18 +12,22 @@ type FilterLogsParams = {
   expandedLines: ExpandedLines;
   expandableRows: boolean;
   failingLine: number | undefined;
+  sectionData: SectionEntry[] | undefined;
+  sectionsEnabled: boolean;
```

Comment:
[nit] I may rename this to `sectioningEnabled`; `sectionsEnabled` potentially sounds like a list of enabled sections.